lua/lean/infoview.lua

```diff
@@ -257,13 +246,7 @@ function M.close_win_wrapper(src_winnr, src_tabnr, close_info, already_closed)
     end
   end
 
-  if close_info then
```

Comment:
The removal of this distinction results in `E855: Autocommands caused commands to abort` when closing the last (Lean) window in a tab with `:close` - I don't think `WinClosed` is designed to be able to handle physically closing other windows. From `:help QuitPre`: `Can be used to close any non-essential window if the current window is the last ordinary window.` No similar support exists for `WinClosed`.